examples/Code_search.ipynb

```diff
@@ -243,121 +148,27 @@
   },
```

Comment:
Now that filepaths are filepaths instead of strings, I believe `r[1].filepath` on line 141 needs to be wrapped in `str()` so it can print.

Otherwise, everything ran great for me. Thanks!

Reply:
Updated in new commit, thanks.